lib/generators/geoblacklight/templates/catalog_controller.rb

```diff
@@ -33,7 +33,9 @@ class CatalogController < ApplicationController
 
     config.show.display_type_field = 'format'
 
-
+    # Custom GeoBlacklight fields which currently map to GeoBlacklight-Schema
+    # v0.3.2
+    config.wxs_identifier_field = 'layer_id_s'
```

Comment:
What are the advantages to making the wxs_identifier_field configurable? How would this be used in the wild?